### ISSUE TITLE
Only x86* is supported

### ISSUE THREAD

**Issue by ggardet** [informatikr/scrypt#8]
Currently, only x86* is supported due to SSE. It would be nice to enable it for other archs too.

**Comment by mark-stopka** [comment]
Agreed! @mimi1vx any idea how much work that would be? Use the C without optimizations on generic platforms, and make SSE2 conditional on non-x86 platforms?

**Comment by DanBurton** [comment]
bump

**Comment by ivb-supercede** [comment]
If this is still important to anyone - it was to us, since we're using `password` - we've got a fork [over here](https://github.com/SupercedeTech/scrypt) that binds to Tarsnap's system library instead.

The Tarsnap library has all the necessary conditional compilation to make it cross-platform, so it can be used on at least some non-x86 machines.

**Comment by rowanG077** [comment]
Running into the same issue. I'm running @ivb-supercede fork with no issues. Honestly the manually copy pasted C code are quite concerning in a crypto library bindings. Would you be willing to take over the package @ivb-supercede? There has been no update in almost 10 years so I doubt you will have a lot of maintenance burden.
